packages/sdk/client/src/services/local-client-services.ts

```diff
@@ -112,6 +142,17 @@ export class LocalClientServices implements ClientServicesProvider {
     });
     await this._host.open(this._ctx);
     this._isOpen = true;
+    // don't set identity tags for non-browser usage, e.g. agents
+    // TODO(nf): fix
```

Comment:
what needs to be fixed?

Reply:
now I remember...

I wasn't sure what the behavior should be when the app isn't known, which is currently the agent, but might be other usages in the future.

I don't think including these sources and not including the origin field will significantly affect the usefulness of the data, but maybe @zhenyasav / @jessmartin have other ideas.